test/powershell/Host/TabCompletion/TabCompletion.Tests.ps1

```diff
@@ -250,6 +260,28 @@ Describe "TabCompletion" -Tags CI {
             Pop-Location
         }
 
+        It "TabCompletion should be case-insensitive for file names on Windows and MacOS" -Skip:($IsLinux) {
+            Push-Location -Path $tempDir
+            $res = TabExpansion2 -inputScript $oneSubDirLowerTest -cursorColumn $oneSubDirLowerTest.Length
+            $res.CompletionMatches.Count | Should BeGreaterThan 0
+            $res.CompletionMatches[0].CompletionText | Should Be ".${separator}$oneSubDirTest"
+
+            $res = TabExpansion2 -inputScript $oneSubDirUpperTest -cursorColumn $oneSubDirUpperTest.Length
+            $res.CompletionMatches.Count | Should BeGreaterThan 0
```

Comment:
Shouldn't we know exactly the number of expected completions?

Reply:
For this fix, it does not matter count (the check is only protection from exception in next line) - the order is important.

Reply:
Since this test created the folder structure, the number of completions should be the same each time, right?